sql/mysql/driver.go

```diff
@@ -46,6 +46,14 @@ func Open(db schema.ExecQuerier) (*Driver, error) {
 	if err := sqlx.ScanOne(rows, &c.version, &c.collate, &c.charset); err != nil {
 		return nil, fmt.Errorf("mysql: scan system variables: %w", err)
 	}
+	if c.tidb() {
+		return &Driver{
+			conn:        c,
+			Differ:      &sqlx.Diff{DiffDriver: &diff{c}},
+			Inspector:   &tinspect{&inspect{c}},
+			PlanApplier: &planApply{c},
+		}, nil
+	}
```

Comment:
Override only the inspector.

Reply:
will do the others as needed in the next PRs